server/services/party/entities/party.ts

```diff
@@ -1,25 +1,22 @@
 import {Column, Entity, Index, PrimaryGeneratedColumn} from 'typeorm'
 
-@Entity({name:'parties'})
-@Index('party', ['partyId', 'name'])
+@Entity({name: 'parties'})
+@Index('party', ['name'])
 export class Party {
     @PrimaryGeneratedColumn()
     id: number
-
-    @Column()
-    partyId: number
     @Column()
     name: string
 
     @Column()
     partyMembers: number
 
     @Column('simple-array')
-    members: string[]
+    members: number[]
```

Comment:
keeping character IDs should be fine, because if we rely on names, character names can potentially change which would break the relationship.